pkg/updater/updater.go

```diff
@@ -195,3 +205,15 @@ func (u *Updater) Start(ctx context.Context) <-chan struct{} {
 
 	return doneChan
 }
+
+// CommitmentTransactionsTarget is a struct that encampsulates the structure of a bundle commitment.
+//
+// The current structure is a comma separated list of transaction hashes.
```

Comment:
The reason I kept it there, was in case we wanted to change the internal representation of the bundle. By keeping it as it's own type, we can basically searlize/desearlize without having to worry about representation in code that shouldn't be concerned by it. Anyhow, since it's premature optimization, I've changed it. Would love to understand if the reasoning above doesn't fit.